environment.ts

```diff
@@ -112,6 +139,36 @@ export type Environment = {
   UNLOCKABLE_CONTENT: boolean
 }
 
+const cache = new LRUCache({
+  max: 100,
+  ttl:
+    typeof window === 'undefined'
+      ? 1000 * 10 // 10 sec
+      : 0, // no ttl, user need to refresh the page
+  fetchMethod: async (key: string) => {
+    invariant(key === 'config', 'Invalid key')
+    const { config } = await request<{ config: RemoteConfig }>(
+      `${
+        process.env.NEXT_PUBLIC_LITEFLOW_BASE_URL || 'https://api.liteflow.com'
+      }/${process.env.NEXT_PUBLIC_LITEFLOW_API_KEY}/graphql`,
```

Comment:
Can we use the `NEXT_PUBLIC_LITEFLOW_API_KEY` as key to be sure to separate the cache per org:
```suggestion
    const { config } = await request<{ config: RemoteConfig }>(
      `${
        process.env.NEXT_PUBLIC_LITEFLOW_BASE_URL || 'https://api.liteflow.com'
      }/${key}/graphql`,
```